packages/zoltan2/test/unit/util/findUniqueGids.cpp

```diff
@@ -509,6 +510,19 @@ int main(int narg, char *arg[])
               << "GO == int" << std::endl;
 #endif
 
+#ifdef HAVE_TPETRA_INT_LONG
+  test1<long>(comm);
+  test2<long>(comm);
+  test3<long>(comm);
+  test4<long>(comm);
+  test5<long>(comm);
+  test6<long>(comm);
+#else
+  if (comm->getRank() == 0) 
+    std::cout << "Skipping long long tests because Tpetra is not build with "
```

Comment:
This comment doesn't quite make sense -- did you mean "Skipping long tests because Tpetra is not built with GO == long"?